### ISSUE TITLE
Support building with Visual Studio 2022

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#13033]
### Is your feature request related to a problem? Please describe.
Visual Studio 2022 is to be released next week. NVDA builds with Visual Studio 2019.

### Describe the solution you'd like
Support building with Visual Studio 2022. We can decide to support 2019 and up, but as soon as appveyor supports 2022, it makes sense to drop 2019 support altogether.

### Additional context
Blocked by a new SCons release. Visual Studio 2022 support is in master but is awaiting a new release.

**Comment by seanbudd** [comment]
AppVeyor now supports this: https://www.appveyor.com/blog/2021/11/10/visual-studio-2022-image/

**Comment by jordo** [comment]
any rough eta on this release?

**Comment by leonardder** [comment]
For SCons, they said today or tomorrow, but that was the day before 
yesterday.

**Comment by leonardder** [comment]
This was reverted by #13387

**Comment by leonardder** [comment]
The fact that the uia library doesn't build on VS2022 is definitely an issue in our build environment. Running msbuild from the vs2022 developer command prompt works just fine.

**Comment by codeofdusk** [comment]
This issue should be reopened, since NVDA doesn't yet support building with VS 2022 build tools. Is this still under investigation and blocked by issues in the UIA remote ops library?

**Comment by leonardder** [comment]
You should be able to install the VS2019 toolset with VS2022 build tools 
as well, I assume?

**Comment by cary-rowen** [comment]
@codeofdusk 
I seem to have the same problem.

@leonardder 
@seanbudd 
Can you investigate this?

**Comment by leonardder** [comment]
Please provide logging output if possible. I have no idea where to 
start. I have VS 2022 enterprise installed on both systems and both 
environments work like a charm as long as I have the VS2019 build tools 
installed.
